src/components/SeatLayout/seatConfig.ts

```diff
@@ -0,0 +1,105 @@
+import { ISeat, ISeatStatus } from '../../api/types/trip';
+import { IDeckLayout, ILayoutConfig } from './types';
+
+export const layoutNames = {
+    volvo25: 'volvo25',
+};
+
+export const layoutConfig: ILayoutConfig = {
+    [layoutNames.volvo25]: {
+        lowerDeck: {
+            rows: 12,
+            columns: 5,
+            aisle: 3,
+            door: { 10: 'R' },
+            seatType: 'seater',
```

Comment:
where is seatType being used

Reply:
As of now seatType is not used anywhere. This can be helpful when we have buses mulitple seatTypes.